docs/book/src/user/quick-start.md

```diff
@@ -101,7 +101,10 @@ The clusterctl CLI tool handles the lifecycle of a Cluster API management cluste
 #### Install clusterctl binary with curl on linux
 Download the latest release; on linux, type:
 ```
-curl -L {{#releaselink gomodule:"sigs.k8s.io/cluster-api" asset:"clusterctl-linux-amd64" version:"1.0.x"}} -o clusterctl
+curl -L {{#releaselink gomodule:"sigs.k8s.io/cluster-api" asset:"clusterctl-linux-amd64" version:"1.x" 
```

Comment:
@joekr I don't think we should drop the minor version, given that the code will be moved in a branch and the book version generated on top of it should match the same branch.
Also, somehow related, before fixing this we should figure out https://github.com/kubernetes-sigs/cluster-api/issues/6017
@sbueringer are there updates from netlify?

Reply:
> Also, somehow related, before fixing this we should figure out https://github.com/kubernetes-sigs/cluster-api/issues/6017
> @sbueringer are there updates from netlify?

I didn't get a reply-mail, maybe @CecileRobertMichon 

I think it should be fine to change it to 1.1.x already though to get the current book fixed. I would prefer if we have a 1.1 version ther enow.

Reply:
yeah I was debating that. I've updated it now to be `1.x.x`

Reply:
Updated